test/integration/test_grpc_service.py

```diff
@@ -42,3 +42,25 @@ def test_grpc_service_stress(http_test_server_fixture):
   counters = http_test_server_fixture.getNighthawkCounterMapFromJson(parsed_json)
   assertGreaterEqual(counters["benchmark.http_2xx"], 5000)
   assertEqual(counters["requestsource.internal.upstream_rq_200"], 4)
+
+
+def run_service_with_args(args):
```

Comment:
Can we add documentation for this helper? E.g.

```py
def run_service_with_args(args):
  """Executes the Nighthawk service with the provided arguments.

  Args:
    args: A string, the command line arguments for the service, e.g. "--foo --bar".

  Returns:
    A tuple in the form (exit_code, output), where exit_code is the code the Nighthawk
    service terminated with and the output is its standard output.
  """
```

Comment about adding documentation applies to all new non-test public functions. We could consider making some of them private instead.

Reply:
I made added this comment, slightly modified, to `run_service_with_args`. And then made this function and others that reuse it private.